lib/anyplayer/players/spotify_mac.rb

```diff
@@ -47,22 +47,21 @@ def album
 
   def playing?
     playing = spotify 'return player state is playing'
-    playing == "true"
+    playing == 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.